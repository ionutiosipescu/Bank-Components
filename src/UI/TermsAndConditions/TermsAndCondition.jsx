import React from "react";
import { Link } from "react-router-dom";
import { Checkbox } from "../Input/Input.style";


function TermsAndCondition() {
  return (
    <Checkbox>
      <input type="checkbox" />
      <p>
        Yes, I agree with the <Link to="#">Terms and Services</Link>
      </p>
    </Checkbox>
  );
}

export default TermsAndCondition;
