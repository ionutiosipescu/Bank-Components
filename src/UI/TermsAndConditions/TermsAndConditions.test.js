import React from "react";
import { shallow, mount, render } from "enzyme";
import TermsAndCondition from "./TermsAndCondition";

it("expect to render TermsAndCondition component", () => {
  expect(shallow(<TermsAndCondition />)).toMatchSnapshot();
});
