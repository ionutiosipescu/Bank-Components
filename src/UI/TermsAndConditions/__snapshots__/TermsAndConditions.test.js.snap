// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render TermsAndCondition component 1`] = `
<styled.div>
  <input
    type="checkbox"
  />
  <p>
    Yes, I agree with the 
    <Link>
      Terms and Services
    </Link>
  </p>
</styled.div>
`;
